employee-rostering-backend/pom.xml

```diff
@@ -55,90 +54,45 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <java.version>1.8</java.version>
-    <version.org.jacoco>0.8.3</version.org.jacoco>
   </properties>
 
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.optaplanner</groupId>
+        <artifactId>optaplanner-bom</artifactId>
+        <version>7.22.0.Final</version>
```

Comment:
Is there a reason why this particular version has been used? I would expect 7.24.0-SNAPSHOT, or if we want to use a final version, then use the latest one - 7.23.0.Final.

Reply:
I think the non-latest version is not a problem at the moment. Master is still on snapshot so if there's a change in optaplanner that breaks employee rostering, we'll know about it.

During the migration it will be convenient to use a final version so that you don't wait for artifacts download each morning.

Reply:
Makes sense, still I would rather use the latest final version, 7.23.0.Final. But it's not a big deal.